contracts/Claims.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity ^0.7.5;
+
+// @notice Used to create Radicle Link identity claims
+contract Claims {
+    /// @notice Emitted on every Radicle Link identity claim
+    /// @param addr The account address which made the claim
+    event Claimed(address indexed addr);
+
+    /// @notice Creates a new claim of a Radicle Link identity.
+    /// Every new claim invalidates previous ones made with the same account.
+    /// The claims have no expiration date and don't need to be renewed.
+    /// If either `format` is unsupported or `payload` is malformed as per `format`,
+    /// the previous claim is revoked, but a new one isn't created.
+    /// Don't send a malformed transactions on purpose, to properly revoke a claim see `format`.
+    /// @param format The format of `payload`, currently supported values:
+    /// - `1` - `payload` is exactly 20 bytes and contains an SHA-1 root hash.
```

Comment:
regarding these references to "root hash", I wonder if we should just remove them or make it clear what it is referring to, since now there is no context to it.

Reply:
Hmm yeah this doc will get outdated as soon as we extend the payload beyond those two things. On the other hand, the docs as-is isn't wrong: the expected payload _is_ the root hash when format = 1

Reply:
yap. maybe say "Radicle Identity root hash"?

Reply:
I've changed it into:
```suggestion
    /// - `1` - `payload` is exactly 20 bytes and contains an SHA-1 Radicle Identity root hash
```

Is that clear enough?